src/coldfront_plugin_api/scim_v2/auth_middleware.py

```diff
@@ -0,0 +1,17 @@
+import os
+import logging
+
+from django_scim.middleware import SCIMAuthCheckMiddleware
+from mozilla_django_oidc.contrib.drf import OIDCAuthentication
+
+logger = logging.getLogger(__name__)
+
+
+class SCIMColdfrontAuthCheckMiddleware(SCIMAuthCheckMiddleware):
+    def process_request(self, request):
+        if not request.user or not request.user.is_staff:
```

Comment:
Do not check for is_staff.

Reply:
When testing, I saw that `request.user` by default is set to an anonymous user, which seems to be the default behavior for [Django authentication ](https://docs.djangoproject.com/en/5.1/topics/auth/default/#authentication-in-web-requests). This means the `not request.user` check always passes. 

I guess I should have wrote `request.user.is_authenticated` instead of `is_staff`.

Reply:
Yes, use is_authenticated